README.md

```diff
@@ -168,6 +169,15 @@ $ java -jar target/release-manager-$(xpath -q -e  "/project/version/text()" pom.
   update-build-config \
   -g snowdrop/spring-boot-bom -o ${GITHUB_TOKEN} -glu ${GITLAB_USER} -glt ${GITLAB_TOKEN} -r 2.4.3  -q Alpha1 -m "DR*"
 ```
+
+## Testing
+
+The unit tests require the providing of 2 parameters to be able to login to the JIRA REST API.
+
+```bash
+$ mvn test -Djboss.jira.user=${JBOSS_JIRA_USER} -Djboss.jira.password=${JBOSS_JIRA_PWD}
```

Comment:
Should we add a maven profile to run such Integration tests (= against a JIRA server) instead and to be sure that the github workflow action will not execute them ?

Reply:
This is what I have finished implementing by the end of last week. I just realized some documentation was missing on the matter so I just added it now. We now have more information on the test profiles in the [Testing](https://github.com/jacobdotcosta/release-manager/tree/int-tests-jira-rest-api#testing) section.

@cmoulliard